docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -122,7 +122,7 @@ The following examples check the value of the **Release** entry to determine whe
 
 01. Check the **Release** REG_DWORD value to determine the installed version. To be forward-compatible, check for a value greater than or equal to the value listed in the [.NET Framework version table](#version_table).
 
-The following example checks the value of the **Release** entry in the registry to find the versions of .NET Framework 4.5-4.8 that are installed:
+The following example checks the value of the **Release** entry in the registry to find the versions of .NET Framework 4.5-4.8 that are installed. Note that, for the examples in this document, you'll need to include the `Microsoft.Win32` using directive at the top of your scripts.
```

Comment:
I can't add a suggestion for some reason, but can you change the sentence you added to:

```

> [!TIP]
> Add the directive `using Microsoft.Win32` at the top of your code file.
```

Reply:
I can't edit since I deleted the fork. Should I make a new PR, or are you able to edit?

Reply:
I'm not able to edit it either. Could you create a new PR?

Reply:
Yes, I'll do that.